src/components/InventoryGroupDetail/GroupTabDetails.js

```diff
@@ -23,83 +24,78 @@ const GroupTabDetailsWrapper = ({
   activeTab,
   hasEdgeImages,
 }) => {
-  const [tab, setTab] = useState(0);
+  const [groupTabKey, setGroupTabKey] = useState(groupTabKeys.systems);
+  const [hybridInventoryTabKey, setHybridInventoryTabKey] = useState(
+    hybridInventoryTabKeys.conventional.key
+  );
 
   const { hasAccess: canViewHosts } = usePermissionsWithContext(
     REQUIRED_PERMISSIONS_TO_READ_GROUP_HOSTS(groupId)
   );
-  const conventionalSystemsContent = useMemo(
+
+  const ConventionalTabTable = useMemo(
     () => (
       <GroupSystems
         groupName={groupName}
         groupId={groupId}
         hostType={hybridInventoryTabKeys.conventional.key}
       />
     ),
-    [groupId, groupName]
+    []
```

Comment:
Why do these need to be removed? And also if there is no specific reason for them to be in a `useMemo` we should maybe take them out and put them into the `return`.